annotation-processing/src/test/java/com/exactpro/epfast/annotation/processing/FastProcessorTest.java

```diff
@@ -52,7 +66,44 @@ void testDuplicateFastTypeNamesAreNotProcessed() {
         JavaSourcesSubject.assertThat(
             JavaFileObjects.forResource("test/DuplicateA.java"),
             JavaFileObjects.forResource("test/DuplicateB.java"))
-            .processedWith(new FastProcessor())
+            .processedWith(fastProcessor)
             .failsToCompile().withErrorContaining("Multiple declarations referring to FastType 'duplicate' are found.");
     }
+
+    @Test
+    void testFindRootPackage() {
```

Comment:
Move these tests to separate class. They are pure unit tests. Keep AnnotationProcessorTests name for them, place compilation tests into AnnotationProcessorCompilationTests